repose-aggregator/src/docs/asciidoc/recipes/client-authentication.adoc

```diff
@@ -0,0 +1,92 @@
+= SSL/TLS Client Authentication
+
+SSL/TLS Client Authentication is being used more and more for communications between different enclaves.
+This addition to the SSL/TLS handshake involves the Client presenting credentials to the Server in the same manner as the Server does to the Client.
+If the credentials presented by the Client are not trusted, then the Server will sever the connection just as the Client would have if the situation was reversed.
+Since a Client initiates contact with the Server, the Server's credentials are simply to validate it is who the Client was trying to contact.
+This is accomplished through Certificate Authorities (CA) and the Trust Hierarchies built into the Public Key Infrastructure (PKI).
+Even though you can optionally add a particular Server's credentials directly into a Client so that it will implicitly trust a particular Server essentially bypassing the distributed trust mechanism in favor of a more direct one, this is the only way to build a relationship for a Client to a Server.
+
+== Generate and Install the Certificate
+
+Regardless of which end of *Repose* you are setting up SSl/LS Client Authentication on you'll need to follow these steps.
+For the purposes of these steps the _client_ will refer to the requesting service for requests inbound to *Repose* if you are setting up for client authentication on the inbound side of *Repose*, and *Repose* itself if you are setting it up for the outbound to origin service side.
+The _server_ will refer to *Repose* if you are setting up for client authentication on the inbound side of *Repose*, and the origin service if you are setting it up for the outbound to origin service side.
```

Comment:
I feel like you can reorganize this bit to improve clarity. My attempt at doing so is below:

> In the following steps, the _client_ will refer to the entity sending the request, and the _server_ will refer to the entity receiving the request. 
> If you are setting up client authentication for requests inbound to Repose, then the entity sending the request is the _client_ and Repose is the _server_.
> If you are setting up client authentication for requests outbound from Repose to the origin service, then Repose is the _client_ and the origin service is the _server_.